app/src/main/AndroidManifest.xml

```diff
@@ -5,6 +5,7 @@
     <uses-permission android:name="android.permission.INTERNET" />
 
     <application
+        android:name=".ExampleApplication"
```

Comment:
how come example application?

Reply:
To use hilt, we need to set the application to hilt and then from hilt to the real application